target/linux/ramips/image/mt7621.mk

```diff
@@ -48,6 +48,20 @@ define Build/iodata-factory
 	fi
 endef
 
+define Build/iodata-mstc-header
+  ( \
+    data_size_crc="$$(dd if=$@ ibs=64 skip=1 2> /dev/null | gzip -c | tail -c 8 | od -An -tx8 --endian little | tr -d ' \n')"; \
+    echo -ne "$$(echo $$data_size_crc | fold -s2 | xargs -I {} echo \\x{} | tr -d '\n')" | \
+      dd of=$@ bs=1 count=8 seek=56 conv=notrunc 2> /dev/null; \
```

Comment:
bs=8 count=1 seek=7 ?
However, I admit it probably won't have a measurable speed effect.

Reply:
fixed

Reply:
(Note that this is not so much a fix, but two different ways to solve the problem.)
But anyway, thanks for changing.